### ISSUE TITLE
Icon: The star icon does not meet contrast for a11y

### ISSUE THREAD

**Issue by seangates** [eBay/skin#184]
![image](https://user-images.githubusercontent.com/105656/39660274-fed2d8d6-4fef-11e8-8b11-37d2fac06ea3.png)

We need to work with the design team to find a way to address the contrast issue. The current gray color (for three of the four icons above) does not meet a11y standards.

**Comment by senthilp** [comment]
@seangates In a design review yesterday there were also concerns about the color of the stars. Apparently, A/B testing doesn't seem favorable for black stars. We need to check with the design team if the original default color (yellow/orange) should be used.

**Comment by seangates** [comment]
So, the response from the design team is as follows:
>**[redacted] [3:07 PM]**
>hey @seangates - after talking with [redacted] it seems like disable states, images, and logos don't legally have to pass a11y. i'm going to start a discussion with the team on this to take another holistic look at our stance... let's keep in touch on this.

Although I don't see these as "disabled". We should still drive to have a color that is contrast compliant, just like text.

**Comment by ianmcburnie** [comment]
Current icons:

<img width="882" alt="screen shot 2019-03-04 at 5 33 20 pm" src="https://user-images.githubusercontent.com/38065/53774502-aa580f80-3ea3-11e9-95ea-79be9198ff1c.png">

They *appear* dark enough now for a white background (would have to verify of course), but the contrast between the two halves of "Star Half" needs to be increased, as it almost looks like a full star to me.

**Comment by seangates** [comment]
Newest, present icons:

![image](https://user-images.githubusercontent.com/105656/65267559-802e0e00-dad2-11e9-8f1c-c5af4d7b0cf5.png)

**Comment by ianmcburnie** [comment]
Yep, closing.
